src/controllers/browser.ts

```diff
@@ -56,18 +57,15 @@ export async function initWhatsapp(
 
     const { userPass, userProxy, addProxy } = options
 
-    // TODO Create own cache version and download it or use HTML from options
-    if (typeof options.webVersion === 'string' && options.webVersion.length) {
+    if (
```

Comment:
whatsappCacheManager/service
 ! return null
this.existsWAVErsionHTML
this.existsWAVErsion